build.yaml

```diff
@@ -1680,6 +1680,25 @@ steps:
     dependsOn:
       - hail_linting_image
       - merge_code
+  - kind: runImage
+    name: compile_hail_213
+    image:
+      valueFrom: base_image.image
+    resources:
+      memory: standard
+      cpu: '2'
+    script: |
+      set -ex
+      cd /io/repo/hail
+
+      export MILLOPTS='--no-daemon' HAIL_BUILD_MODE='CI' SCALA_VERSION='2.13'
```

Comment:
Is there a need to set `SCALA_VERSION` here given you're explicitly naming the version in the mill target?

Reply:
Nope